build.gradle

```diff
@@ -322,25 +344,3 @@ uploadArchives {
         System.out.println("Uploading version $version")
     }
 }
-
-//update static web docs
-task copyJavadoc(dependsOn: 'javadoc', type: Copy) {
```

Comment:
@vdauwera We've discussed this in the past, but just want to confirm: once we commit this branch, the only doc generation capability we'll have for Picard will be the equivalent of gatkDoc, which will generate Barclay-style syntax, and will require a the addition of @DocumentedFeature annotations).